test-data/unit/check-varargs.test

```diff
@@ -381,15 +381,17 @@ main:4: error: Argument 2 to "f" has incompatible type *List[A]; expected "B"
 main:5: error: Argument 3 to "f" has incompatible type *List[A]; expected "B"
 main:6: error: Argument 1 to "f" has incompatible type *"Tuple[A, A, B]"; expected "B"
 
-[case testVarArgsAfterKeywordArgInCall1]
+[case testVarArgsAfterKeywordArgInCall1-skip]
+# see: mypy issue #2729
```

Comment:
I'm confused, that issue *claims* to be about Python 2, but this test (or the next) is not about Python 2 IIUC. What am I missing?